plugins/privacy-view.js

```diff
@@ -32,9 +32,10 @@ privacyView.toggle = () => {
   privacyView.is_active = !privacyView.is_active;
   if (window.isSmartphone()) {
     IITC.toolbox.updateButton('privacytoggle', { label: privacyView.text[privacyView.is_active] });
+    document.body.classList.toggle('privacy_active');
```

Comment:
This toggle is the same for mobile and desktop, it is better to put it after line 32